public/public/assets/css/style.css

```
@@ -569,6 +569,10 @@ section {
   padding-bottom: 30px;
 }
 
+.portfolio .marketplaceTitle {
```

Put CSS in corresponding React component to follow best practices
```suggestion
.portfolio .marketplace-title {
```